src/RuleSet/DeclarationBlock.php

```diff
@@ -63,7 +61,7 @@ public static function parse(ParserState $parserState, ?CSSList $list = null)
                 if (!$parserState->comes('}')) {
                     $parserState->consumeUntil('}', false, true);
                 }
-                return false;
+                return null;
```

Comment:
We should cover this with a (functional) test.

Reply:
Done.